lib/internal/validators.js

```diff
@@ -152,7 +156,17 @@ function validateEncoding(data, encoding) {
                                     `is invalid for data of length ${length}`);
   }
 
-  // TODO(bnoordhuis) Add BASE64 check?
+  // Rules for BASE64:
+  // 1. Data's length % 4 === 0.
+  // 2. Each character SHOULD BE either an English Alphabet in either lower
+  // or upper case, or "=","/","+".
+  // 3. The last two characters could be "=".
+  // For more, please see https://tools.ietf.org/html/rfc4648#section-5
+  if (normalizeEncoding === 'base64' && (!base64CheckReg.test(data))) {
```

Comment:
Shouldn't `normalizeEncoding` be `normalizedEncoding` (note normalize vs. normalized)?